yt/yt/core/misc/unittests/error_ut.cpp

```diff
@@ -421,6 +421,48 @@ TEST(TErrorTest, SerializationDepthLimit)
     }
 }
 
+TEST(TErrorTest, DoNotDuplicateOriginalErrorDepth)
+{
+    constexpr int Depth = 20;
+    auto error = TError(TErrorCode(Depth), "error");
+    for (int i = Depth; i >= 2; --i) {
+        error = TError(TErrorCode(i), "error") << std::move(error);
+    }
+
+    auto errorYson = ConvertToYsonString(error);
+    error = ConvertTo<TError>(errorYson);
+
+    // Due to reserialization error already contains "original_error_depth" attribute.
```

Comment:
A comma after "reserialization" would eliminate ambiguity with "Due to reserialization error, <my mind stumbles here>".